blog/2024-10-30-openrefine-developer-role.md

```diff
@@ -0,0 +1,63 @@
+  * Improve reproducibility features (per EOSS-5 grant requirements).
+  * Maintain Wikimedia-related extensions, optimizing for stability and user experience (pending grant approval).
+  * Implement quality improvements through refactoring, optimization, and code maintenance.
+  * Improvements through refactoring, optimization, and code maintenance.
+* **Community Support & Engagement**
+  * Participate in community forums to support OpenRefine users and volunteers, fostering a positive and engaged user community.
+* **Pull Request Reviews & Ticket Triage**
+  * Once familiar with the codebase, actively engage with community-submitted pull requests, providing constructive feedback and participating in collaborative review processes.
+  * Assist with ticket triage to identify and prioritize key issues and opportunities.
+* **Progressive Onboarding into Release Management & Security Updates**
+  * Upon receiving commit privileges, assist in merging pull requests to support the workflow of community contributions.
+  * ake on Release Manager responsibilities to prepare and publish new releases, coordinating with the community on release planning.
+  * With further onboarding, address security advisories and implement required updates and patches to maintain OpenRefine’s security.
+* **Internship Program Participation**
+  * Support programs such as Google Summer of Code (GSoC) and Outreachy, mentoring interns, reviewing their contributions, and fostering a supportive learning environment.
+
+## Work Environment & Reporting Structure
+
+This is a fully remote role with flexible working hours, allowing the developer to organize their time freely over the month. The OpenRefine Developer will primarily work with volunteers, including members of the Advisory Committee and most technical contributors. The OpenRefine community collaborates asynchronously through public forums and GitHub, accommodating the diverse time zones and work patterns of contributors. Occasional travel may be expected to attend relevant conferences and meetings, enabling valuable in-person engagement with the OpenRefine community and the broader open-source ecosystem.
+
+The developer will report to the OpenRefine Advisory Committee, which will guide project goals and prioritization. For technical supervision and mentorship, the developer will collaborate closely with the OpenRefine Committer Group—comprising both volunteer contributors and paid contractors—to facilitate their integration with the codebase and OpenRefine’s development standards.
+
+This role requires a self-motivated, independent approach, with minimal direct supervision. Weekly check-ins with the Advisory Committee and Project Manager will provide alignment with OpenRefine’s broader objectives and ongoing support as needed.
+
+## Skills 
+* **Java Proficiency**: As OpenRefine’s core codebase is Java, strong experience in Java is crucial.
```

Comment:
Since we have a "Nice to have" below, should this bullet point list start with "Required"?